test_bot/config.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+
+try:
+    import dotenv
+except ModuleNotFoundError:
+    print("Not loading .env")
+else:
+    dotenv.load_dotenv()
+
+
+class Config:
+    token = os.environ.get("BOT_TOKEN")
+    prefix = os.environ.get("PREFIX", ".")
+    cogs_folder = os.environ.get("COGS_FOLDER", "cogs")
+    auto_reload = os.environ.get("AUTO_RELOAD", "false") == "true"
+    sync_permissions = os.environ.get("SYNC_PERMISSIONS", "true") == "true"
+    sync_commands_debug = os.environ.get("SYNC_COMMANDS_DEBUG", "true") == "true"
+    test_guilds = (
+        [int(x.strip()) for x in test_guilds.split(",")]
+        if (test_guilds := os.environ.get("TEST_GUILDS", None))
+        else None
+    )
+    debug = os.environ.get("DEBUG", "false") == "true"
+    debug_loggers = os.environ.get("DEBUG_LOGGERS")
+    log_to_file = bool(os.environ.get("LOG_TO_FILE", "false").lower() == "true")
```

Comment:
more explicit

Reply:
What about moving [`.ext.commands.converter._convert_to_bool`](https://github.com/DisnakeDev/disnake/blob/3a65209481add1113238b439135531413746644d/disnake/ext/commands/converter.py#L1147) to a utils method and using that?
In any case, the `bool` call is unnecessary since `str.__eq__` already returns a bool

Reply:
If we moved that, it would be a seperate pr, and additionally, the test bot (imo) shouldn't use anything undocumented or internal if it can be helped.

Reply:
https://github.com/DisnakeDev/disnake/pull/247/commits/21ce44f9e4186967edb7f3895676d00200dd6a28